hoomd/pytest/test_state.py

```diff
@@ -223,6 +223,30 @@ def test_thermalize_angular_momentum(simulation_factory,
         assert K > expected_K * 3 / 4 and K < expected_K * 4 / 3
 
 
+def test_zero_particle_velocity_angmom():
+    snapshot = hoomd.Snapshot()
+    snapshot.configuration.box = (10, 10, 10, 0, 0, 0)
+    snapshot.particles.N = 4
+    snapshot.particles.types = ['A']
+    snapshot.particles.body[:] = [0, 0, 2, 2]
+    snapshot.particles.moment_inertia[:] = [[1, 1, 1]] * 4
```

Comment:
Remember to check if the process is on rank 0 before updating the snapshot's particle data.